osu.Game.Rulesets.Osu/UI/OsuPlayfield.cs

```diff
@@ -158,6 +158,7 @@ private void load(OsuRulesetConfigManager config, IBeatmap beatmap)
             RegisterPool<Spinner, DrawableSpinner>(2, 20);
             RegisterPool<SpinnerTick, DrawableSpinnerTick>(10, 200);
             RegisterPool<SpinnerBonusTick, DrawableSpinnerBonusTick>(10, 200);
+            RegisterPool<SpinnerHealthTick, DrawableSpinnerHealthTick>(10, 200);
```

Comment:
There's no way this is high enough? At least with how you have this implemented now.

And no, I'm not saying to make it higher, I'm pointing out a probable issue with this implementation.

Reply:
Currently this accounts for 24 seconds of post-bonus max spinning (current rate in hp_ticks_per_minute makes this work out to about 8 health ticks per second, can be adjusted), which I figured was a fairly reasonable starting point.